ui/browser/reducers/main-reducers.js

```diff
@@ -55,6 +57,9 @@ export default function basic(state = initialState, action) {
     case types.SET_PAGE_AREA_VISIBILITY:
       return setPageAreaVisibility(state, action.visible);
 
+    case types.SET_WEBVIEW_SCROLL:
```

Comment:
nit: our React class name is `WebView`.  Why are these two not consistent?  (Google appears to use WebView consistently, at least on Android.)